From a96c210c4fa7b0d2118767b42a6fa3e9abbdf0f9 Mon Sep 17 00:00:00 2001
From: Andrzej Hajda <andrzej.hajda@wp.pl>
Date: Mon, 8 Oct 2012 21:28:21 +0200
Subject: [PATCH 1/9] build: Removed python dependency from ldbsamba

Signed-off-by: Andrzej Hajda <andrzej.hajda@wp.pl>
---
 lib/ldb-samba/wscript_build |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/ldb-samba/wscript_build b/lib/ldb-samba/wscript_build
index 63ff5b1..62a249c 100644
--- a/lib/ldb-samba/wscript_build
+++ b/lib/ldb-samba/wscript_build
@@ -8,7 +8,7 @@ bld.SAMBA_LIBRARY('ldbsamba',
                   source='ldif_handlers.c',
                   autoproto='ldif_handlers_proto.h',
                   public_deps='ldb',
-                  deps='samba-security ndr NDR_DRSBLOBS NDR_DNSP ldbwrap samdb-common SAMDB_SCHEMA tdb pyldb-util errors',
+                  deps='samba-security ndr NDR_DRSBLOBS NDR_DNSP ldbwrap samdb-common SAMDB_SCHEMA tdb errors',
                   private_library=True
                   )
 
-- 
1.7.9.5

